pyproject.toml

```diff
@@ -21,15 +21,13 @@ dependencies = [
     "dynaconf>=3.1.2,<4.0",
     "fsspec>=2021.4",
     "gitpython>=3.0",
-    "importlib_metadata>=3.6,<5.0; python_version < '3.8'",  # The "selectable" entry points were introduced in `importlib_metadata` 3.6 and Python 3.10. Bandit on Python 3.7 relies on a library with `importlib_metadata` < 5.0
     "importlib-metadata>=3.6,<7.0; python_version >= '3.8'",
     "importlib_resources>=1.3,<7.0",  # The `files()` API was introduced in `importlib_resources` 1.3 and Python 3.9.
     "jmespath>=0.9.5",
     "more_itertools>=8.14.0",
     "omegaconf>=2.1.1",
     "parse>=1.19.0",
     "pip-tools>=6.5",
-    "pluggy>=1.0,<1.3",  # TODO: Uncap when dropping Python 3.7 support, see https://github.com/kedro-org/kedro/issues/2979
```

Comment:
This causes `nModuleNotFoundError: No module named \'pluggy\'`

Reply:
I will set the pin as pluggy >1.3 since it comes with a breaking API change. pluggy doesn't come with much dependencies so I think it's ok to pin a relative new version.